ophyd/pseudopos.py

```diff
@@ -105,20 +149,184 @@ def _started_moving(self, value):
         pass
 
     def move(self, pos, **kwargs):
+        '''Move this pseudo axis to a specific position.
+
+        See `PseudoPositioner.move_single` for more information.
+
+        Parameters
+        ----------
+        pos : float
+            Position to move to
+        kwargs : dict
+            Passed onto parent.move_single()
+        '''
+        self._target = pos
         return self._parent.move_single(self, pos, **kwargs)
 
+    def read(self):
+        d = OrderedDict()
+        d[self.name] = {'value': self.position,
+                        'timestamp': time.time()}
+        return d
+
+    def describe(self):
+        desc = OrderedDict()
+        desc[self.name] = {'dtype': 'number',
+                           'shape': [],
+                           'source': 'computed',
```

Comment:
bikeshed: might want to make this controllable via init.  ex, shove the formula in here?